lib/exposes.js

```diff
@@ -468,7 +468,7 @@ module.exports = {
         humidity: () => new Numeric('humidity', access.STATE).withUnit('%').withDescription('Measured relative humidity'),
         illuminance: () => new Numeric('illuminance', access.STATE).withDescription('Raw measured illuminance'),
         illuminance_lux: () => new Numeric('illuminance_lux', access.STATE).withUnit('lx').withDescription('Measured illuminance in lux'),
-        keypad_lockout: () => new Lock().withState('keypad_lockout', '1', '0', 'Enables/disables physical input on the device', a.STATE_SET),
+        keypad_lockout_lock1: () => new Lock().withState('keypad_lockout', 'lock1', 'unlock', 'Enables/disables physical input on the device', a.STATE_ALL),
```

Comment:
Why rename this to `keypad_lockout_lock1`?

Reply:
In case we encounter  another combo say lock2 and unlock in the future. I can rename it back though.

Reply:
I think it's good to rename it back, we can always change in the future